charts/castai-cluster-controller/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: castai-cluster-controller
 description: Cluster controller is responsible for handling certain Kubernetes actions such as draining and deleting nodes, adding labels, approving CSR requests.
 type: application
-version: 0.62.0
-appVersion: "v0.44.0"
+version: 0.63.0
```

```suggestion
version: 0.64.0
```
Because I just merged my PR where I upgraded the chart to `0.63.0`